tools/execute-selenium-tests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_HOME="$CURRENT_PATH/.."
+FOXBOX_BINARY="$PROJECT_HOME/target/debug/foxbox"
+
+DEBUG_HOST=${BOX_LOCAL_NAME:-'localhost'}
+DEBUG_PORT=${BOX_PORT:-3331}
+DEBUG_PROFILE="$HOME/.local/share/foxbox-debug"
+
+# clean up residual profile
+rm -rf ${DEBUG_PROFILE}
+
+# build and launch foxbox daemon
+pushd ${PROJECT_HOME}
+cargo build
+$FOXBOX_BINARY -l ${DEBUG_HOST} -p ${DEBUG_PORT} --disable-tls --profile ${DEBUG_PROFILE} &
```

Comment:
Since `cargo run` first builds and then forks the foxbox daemon process, it is harder to get the PID of the actual foxbox process (so we can kill it later). We also then have to explicitly wait for the daemon to spin up (using something like netcat) before continuing to the next line. Doing it this way was just cleaner.